src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(toFileName, content);
+    }
+
+    private String readFile(String fileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] info = line.split(",");
+                if (info[OPERATION_TYPE].equals(BUY)) {
+                    buyAmount += Integer.parseInt(info[AMOUNT]);
+                } else if (info[OPERATION_TYPE].equals(SUPPLY)) {
+                    supplyAmount += Integer.parseInt(info[AMOUNT]);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File not found " + e);
+        }
+        int resultAmount = supplyAmount - buyAmount;
+
+        StringBuilder result = new StringBuilder()
+                .append(SUPPLY).append(",").append(supplyAmount).append(System.lineSeparator())
+                .append(BUY).append(",").append(buyAmount).append(System.lineSeparator())
+                .append("result,").append(resultAmount);
```

Comment:
You are using a magic string `result` in your code. It's better to declare it as a constant. The name of the constant should display this string's purpose. This is a code quality issue.